packages/shared/src/components/buttons/common.ts

```diff
@@ -209,7 +209,7 @@ export const useGetIconWithSize = (
       size: icon.props?.size ?? buttonSizeToIconSize[size],
       className: classNames(
         icon.props.className,
-        !iconOnly && '!h-6 !w-6 text-base',
+        !iconOnly && 'text-base',
```

Comment:
I'm 99% this will break something now...
Can you maybe check when it was introduced and for what output.

I can't recall but the icons should be based on the button size -1 in most cases.

Reply:
Also sifted through the codebase, there were only around 2 or 3 implementations where this will be applied. Also a broken example is the Plus button. The size should have been w-7 h-7 but is being overwritten by the unnecessary important classes.

![image](https://github.com/user-attachments/assets/96bcab56-a56f-49e3-a290-961d566664c5)

Before:

![Screenshot 2025-01-28 at 9 59 18 PM](https://github.com/user-attachments/assets/ff79b7df-363c-4604-bc48-1dbb173e4f24)